app/models/wizards/steps/membership_terminated_info.rb

```diff
@@ -12,6 +12,6 @@ def valid?
   end
 
   def termination_date
-    wizard.person.sac_membership.stammsektion_role.end_on
+    wizard.person.sac_membership&.stammsektion_role&.end_on
```

Comment:
Why is the save access needed? A person must always have an `sac_membership` to call the termination wizard, right?

Reply:
I had an error where this helped to troubleshoot, but will remove now